wiki/People/Global_Moderation_Team/ko.md

```diff
@@ -1,15 +1,12 @@
----
-outdated: true
----
 Global Moderation Team
 ==================
 
-**GMT**로 흔히 알려져 있는 **Global Moderation Team**은 주로 플레이어에 대한 중재, 관리자들과 커뮤니티에 관련된 사안에 중점을 두고 활동하는 osu!Staff에 속한 구성원입니다.
+**GMT** 로 흔히 알려져 있는 **Global Moderation Team** 은 주로 플레이어에 대한 중재, 관리자들과 커뮤니티에 관련된 사안에 중점을 두고 활동하는 osu!Staff에 속한 구성원입니다.
```

Comment:
Whitespace between English characters and Korean characters is not necessary to be added, in this case.